client/components/orderHistory.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import {connect} from 'react-redux'
+import {getMyHistory} from '../store'
+
+class OrderHistory extends React.Component {
+  componentDidMount() {
+    this.props.getMyHistory()
+  }
+  render() {
+    console.log(this.props.orders)
+    const orders = this.props.orders || []
+    return (
+      <div>
+        {orders.length ? (
```

Comment:
The pending orders will be in the cart page so I think u may want to filterout those or u already done that in getMyHistory in redux?